packages/core/directives/polygon.ts

```diff
@@ -224,18 +229,35 @@ export class AgmPolygon implements OnDestroy, OnChanges, AfterContentInit {
   }
 
   private _addEventListeners() {
+    const checkForPathsChange = () => {
+      this._polygonManager.getPolygonPath(this)
+        .then((paths: Array<LatLngLiteral>) => {
+          let arePathsChanged: boolean = false;
+          if (this.paths.length !== paths.length) {
+            this.paths = paths;
+            arePathsChanged = true;
+          } else if (JSON.stringify(this.paths) !== JSON.stringify(paths)) {
```

Comment:
this is simply, but seems horribly slow. Also, how does accuracy and rounding play into this comparison?